R/webshot.R

```diff
@@ -117,7 +118,8 @@ webshot <- function(
   delay = 0.2,
   zoom = 1,
   useragent = NULL,
-  max_concurrent = getOption("webshot.concurrent", default = 6)
+  max_concurrent = getOption("webshot.concurrent", default = 6),
+  verbose = TRUE
```

Comment:
I think it'd be worth controlling this via a global option.

```suggestion
  quiet = getOption("webshot.quiet", default = FALSE)
```